lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs

```diff
@@ -164,6 +169,7 @@ fromMockPrivKey s = (k, Hash (B8.pack s))
 toMockPrivKey :: (Key purpose XPrv, Hash "encryption") -> MPrivKey
 toMockPrivKey (_, Hash h) = B8.unpack h
 
+-- | Mock representation of a DBLayer
```

Comment:
Similarly to the comment above, would recommend turning this into a Haddock hyperlink, like so:
```hs
 -- | Mock representation of a 'DBLayer'.
```

Reply:
Fixed :+1: